src/support/azext_support/custom.py

```diff
@@ -181,11 +175,10 @@ class SupportTicketsNoSubscriptionList(
 
         @property
         def query_parameters(self):
-            from azure.cli.core.aaz import AAZUndefined
 
             parameters = super().query_parameters
             args = self.ctx.args
-            if args.filter == AAZUndefined and args.pagination_limit == AAZUndefined:
+            if has_value(args.filter) and has_value(args.pagination_limit):
```

Comment:
You have inverted the logic in your PR. The initial logic was if filter and pagination is undefined then check and now you have changed it to if filter and pagination has value then check. That means it will basically do the opposite of what it is supposed to do now